crypto/fipsmodule/bn/rsaz_exp_x2.c

```diff
@@ -347,74 +358,70 @@ int rsaz_mod_exp_x2_ifma256(uint64_t *out,
         int exp_chunk_no = exp_bit_no / 64;
         int exp_chunk_shift = exp_bit_no % 64;
 
-        uint64_t red_table_idx_0, red_table_idx_1;
-
-        /*
-         * If rem == 0, then
-         *      exp_bit_no = modlen - exp_win_size
-         * However, this isn't possible because rem is { 1024, 1536, 2048 } % 5
-         * which is { 4, 1, 3 } respectively.
-         *
-         * If this assertion ever fails the fix above is easy.
-         */
-        assert(rem != 0);
-
-        /* Process 1-st exp window - just init result */
-        red_table_idx_0 = expz[exp_chunk_no + 0 * (exp_digits + 1)];
-        red_table_idx_1 = expz[exp_chunk_no + 1 * (exp_digits + 1)];
-
-        /*
-         * The function operates with fixed moduli sizes divisible by 64,
-         * thus table index here is always in supported range [0, EXP_WIN_SIZE).
-         */
-        red_table_idx_0 >>= exp_chunk_shift;
+        uint64_t red_table_idx_1, red_table_idx_2;
+
+	// `rem` is { 1024, 1536, 2048 } % 5 which is { 4, 1, 3 }
+        // respectively.
+        //
+        // If this assertion ever fails the fix above is easy.
```

Comment:
```Suggestion
        // If this assertion ever fails then we should set this easy fix
        // exp_bit_no = modlen - exp_win_size
```
Is that what's intended? Because the change removed "the fix above".